orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/AtlasQueueMonitor.kt

```diff
@@ -176,4 +206,13 @@ class AtlasQueueMonitor
    */
   private val MessageNotFound.counter: Counter
     get() = registry.counter("queue.message.notfound")
+
+  private fun hasBeenAroundAWhile(execution: Execution): Boolean =
+    Instant.ofEpochMilli(execution.buildTime!!)
+      .isBefore(clock.instant().minus(10, MINUTES))
```

Comment:
Would like to see this threshold configurable. I imagine we'll want to tweak this as we go along.

Reply:
Yep